mkdocs.yml

```diff
@@ -11,52 +11,70 @@ markdown_extensions:
 pages:
 - Home: index.md
 - Basics:
-  - Developer Community: basics/community.md
-  - Prerequisite skills: basics/skills.md
-  - Requirements: requirements.md
-  - Planning your project: basics/planning.md
-  - Build: build.md
-  - Customize: customize.md
-  - Extend: extend.md
-  - Develop: develop.md
-  - Testing: testing.md
-  - Writing Documentation: documentation.md
-- Development Environment:
-  - Debugging: dev-tools/debugging.md
-- APIv3:
-  - API Intro: api/general.md
-  - API Usage: api/usage.md
-  - API Actions: api/actions.md
-  - API Parameters: api/params.md
+  - Developer Community: basics/community.md    # page-tree = DONE
+  - Prerequisite Skills: basics/skills.md       # page-tree = DONE
+  - Planning Your Project: basics/planning.md   # page-tree = DONE
+- Documentation:
```

Comment:
Yeah, I can see the logic here, but I'd kinda like to leave the Documentation up at front because 

1. it helps order skills in the guide from easy to hard, (e.g. contributing to docs is a great way to get started, like what I'm doing right now)
1. helps encourage developers to get in the habit of documenting early
1. doc will be a very relevant thing in the coming year as CiviCRM works to transition more docs to MkDocs
1. I definitely don't want to put it at the *very* end because that's where the hooks are going, and with "option 4" for the hooks nothing should go below the hooks because they take up so much space in the menu. 

I'm open to moving this section down, but I'd rather leave it as-is for now, since it's something I have an opinion about, and then discuss more in a dedicated issue before making the change you request. Is that okay? If others feel strongly I'll bow out. But I'd like to push this forward for now and sort this detail out later.